h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -923,14 +924,17 @@ void defaultGBMs(Key<Grid> gridKey) {
     stackedEnsembleParameters._valid = (getValidationFrame() == null ? null : getValidationFrame()._key);
     stackedEnsembleParameters._keep_levelone_frame = true; //TODO Why is this true? Can be optionally turned off
     // Add cross-validation args
+    //TODO: cf. comment in Metalearner; assigning some buildSpec params directly to _metalearner_parameters
+    //  avoid having us to redefine defaults once more here, as well as the duplicate logic here and in  Metalearner
```

Comment:
@ledell, FYI: I have a branch proposing to remove all _metalearner_ params to keep onlythe more generic _metalearner_parameters, and everything seems to work fine.
also noticed that this change was planned somehow according to metalearner_algorithm page of user guide.
I can take associated PUBDEV-5281 if you're interested.